requirements.txt

```diff
@@ -1,2 +1,3 @@
 ipynbname
 numpy
+setuptools_scm
```

Comment:
setuptools_scm is only needed when building the package, we may don't need to put it here. And also maybe remove `numpy`

Reply:
Ok. Then, I'll remove `setuptools_scm` and `numpy`

Reply:
I removed `setuptools_scm` and `numpy` in [75e9e0b](https://github.com/haiiliin/abqpy/pull/1068/commits/75e9e0b587e4b78957d4e477e256ce701fc9e345) but returned `numpy` in [f668b9c](https://github.com/haiiliin/abqpy/pull/1068/commits/f668b9c87ee2fda8c92754fe74a5ab40bebd1f94) because of the PR checkings.

Reply:
It required by test_odb, sorry, my bad.